### ISSUE TITLE
Batch size during training

### ISSUE THREAD

**Issue by aycatakmaz** [astra-vision/PaSCo#18]
Hi,

Thanks a lot for this super cool work!

I have a quick question about the batch size. It appears to me that the train_kitti360.py script sets up the KITTI360DataModule, which uses `collate_fn_simple` in the dataloaders it sets up according to the following line https://github.com/astra-vision/PaSCo/blob/d233f53c77621b735eef2721a128a9cc86f727bf/pasco/data/kitti360/kitti360_dm.py#L184  This collate function (KITTI360DataModule seems to use the one from SemanticKITTI) appears to simply take the first element in the batch according to https://github.com/astra-vision/PaSCo/blob/d233f53c77621b735eef2721a128a9cc86f727bf/pasco/data/semantic_kitti/collate.py#L7

I also saw that the step function explicitly sets the batch size to 1. 
https://github.com/astra-vision/PaSCo/blob/d233f53c77621b735eef2721a128a9cc86f727bf/pasco/models/net_panoptic_sparse_kitti360.py#L267

Does this mean that the training code does not support a batch size>1 at the moment? It appears to me that even if I increase the batch size, the loss computation functions such as loss_mask appear to assume a batch size of 1, if I understand correctly.

Would it be possible for me to get some clarification on this? Thank you very much once again for this great work!

Best,
Ayca

**Comment by anhquancao** [comment]
Thank you for your interest! Yes, the dataloader only supports batch_size = 1 because in the get_item function below, I duplicate the item to create one input per subnet.

https://github.com/astra-vision/PaSCo/blob/d233f53c77621b735eef2721a128a9cc86f727bf/pasco/data/kitti360/kitti360_dataset.py#L108

**Comment by aycatakmaz** [comment]
Thank you so much for the quick response!
